packages/mongodb-dialects/java-driver/src/main/kotlin/com/mongodb/jbplugin/dialects/javadriver/glossary/JavaDriverDialectParser.kt

```diff
@@ -543,6 +645,44 @@ object JavaDriverDialectParser : DialectParser<PsiElement> {
         return typeOfFirstArg != null && typeOfFirstArg.equalsToText(SESSION_FQN)
     }
 
+    private fun parseComputedExpression(element: PsiElement, createsNewField: Boolean = true): HasValueReference<PsiElement> {
+        val (constant, value) = element.tryToResolveAsConstant()
+        return HasValueReference(
+            when {
+                constant && value is String -> HasValueReference.Computed(
+                    element,
+                    type = ComputedBsonType(
+                        BsonAny,
```

Comment:
```suggestion
                        BsonString,
```

Reply:
This I think should be a BsonString because the final expression is written as a String.

Reply:
Yeah but the underlying type of the expression is any possible type right? It's just convenient that MongoDB uses a string for the expression language, but the type is not a string right?